src/components/Button/styled/button.js

```diff
@@ -131,15 +107,17 @@ const StyledButton = styled.button.attrs(props => {
             &:hover,
             &:focus,
             &:active {
-                background-color: ${props.brandDarkColor};
-                border-color: ${props.brandDarkColor};
+                background-color: ${props.palette.brand.dark};
+                border-color: ${props.palette.brand.dark};
                 color: ${brandDarkContrastText};
             }
 
             &[disabled] {
-                background-color: ${props.isLoading ? props.brandMainColor : COLOR_GRAY_1};
-                border-color: ${props.isLoading ? props.brandMainColor : COLOR_GRAY_1};
-                color: ${COLOR_GRAY_2};
+                background-color: ${props.isLoading ? props.palette.brand.main : COLOR_GRAY_1};
```

Comment:
we should use some color from the theme for COLOR_GRAY_1

Reply:
i miss that one